### DISCUSSION TITLE
Command line manual

### DISCUSSION THREAD

**Discussion by ReIaxo** [enzo1982/freac#315] [Help & Support] [answered]
Hi, i have written a little java program to copy the entries of m3u playlists. Now i need to convert some files to mp3 while keeping the metadata.

I already like to use freac from the GUI, but now i want to automate a little.

Can I call freac from the command line? Maybe someone can can provide me an example how i can pass source file, target file and encoder or something like this? Or point me to the right part of the documentation? Thank you in advance!

**Comment by enzo1982** [comment, marked as answer]
Hi, fre:ac comes with a dedicated command line version called freaccmd.

On Windows, you can find the `freaccmd.exe` in the fre:ac application folder (except for the Microsoft Store version of fre:ac). On Linux, you can run the AppImage and pass `freaccmd` as the first argument to invoke the command line version.

Running freaccmd without arguments will print a help page listing the available command line options and encoders. Running `freaccmd -h <encoder>` prints the additional options for the given encoder, e.g. `freaccmd -h lame` prints the encoder-specific options for the LAME MP3 encoder.

Here are some example invocations:

- `freaccmd song.flac`
Convert song.flac to song.mp3 (MP3 being the default output format)

- `freaccmd -e fdkaac song.flac`
Convert song.flac to song.m4a using the FDK-AAC encoder

- `freaccmd -e opus song.flac -o output.opus`
Convert song.flac to output.opus using the Opus encoder

- `freaccmd -e opus song1.flac song2.flac -p "<track> <artist> - <title>"` 
Convert song1.flac and song2.flac and name the output files using the given pattern

- `freaccmd -e opus song1.flac song2.flac -o output.opus`
Convert song1.flac and song2.flac and concatenate them into output.opus
